harness/helpers.ts

```diff
@@ -291,7 +291,12 @@ export class LocalTestClient extends BaseTestClient {
             false,
             this.clientId,
             this.sdkKey,
-            { baseURLOverride: `${getMockServerUrl()}/client/${this.clientId}`, ...options }
+            {
+                eventsAPIBaseURL: `${getMockServerUrl()}/client/${this.clientId}`,
```

Comment:
are you changing the names of these properties on C# to match?

Reply:
If these are the ones we go with I will